pkgs/os-specific/linux/minimal-bootstrap/mes/default.nix

```diff
@@ -222,7 +222,6 @@ let
     mkdir -p ''${out}/bin
 
     ${srcPost.bin}/bin/mes-m2 -e main ${srcPost.bin}/bin/mescc.scm -- \
-      --base-address 0x08048000 \
```

Comment:
In other places you switched this to `--base-address ${baseAddress}`, is it not required here?

Reply:
Apparently not? I left it out and it seems to compile fine. Other uses of `mes-m2` also seem to use it optionally and I couldn't see a consistent pattern. It would be more convenient to not have to use it and leave the lookup table internal to `stage0-posix` but that's just a selfish wish :p